src/Components/NavBar/NavItem.tsx

```diff
@@ -115,10 +117,31 @@ export const NavItem: React.FC<NavItemProps> = ({
         </Sans>
       </Link>
 
+      {/* Very hacky fix to prevent mouse out from triggering a close on the
+          first Artworks nav item. Create a box, position and extend it out a
+          ways and then hide it.
+
+          FIXME: Come up with a better way to do this
+       */}
+      {isFullScreenDropDown && label === "Artworks" && showMenu && (
+        <Box
+          position="absolute"
+          style={{
+            background: "green",
+            zIndex: 1,
+            width: 300,
+            height: 50,
+            marginTop: -50, // using margin because we can't use relative positioning here
+            marginLeft: -220,
+            opacity: 0,
+          }}
+        />
+      )}
+
```

Comment:
I guess this wouldn't be necessary for the Artists drop down as well?

Reply:
Nope, because it makes sense that if the user mouses over to the Artworks nav that a new nav would appear. Not so with angularly mousing over a wide search bar.

Reply:
Makes sense!

Reply:
Crafty! 😄